antora-ui-camel/src/js/05-nav-search.js

```diff
@@ -6,9 +6,18 @@ document.addEventListener('DOMContentLoaded', function () {
   if (!navSearch) {
     return
   }
-
+  var navToggle = document.querySelectorAll('.nav-item-toggle')
   var navLinks = document.querySelectorAll('.nav-link')
   navSearch.addEventListener('keyup', function () {
+    if (navSearch.value === '') {
+      for (var b = 0; b < navToggle.length; b++) {
```

Comment:
```suggestion
      for (let i = 0; i < navToggle.length; i++) {
```

Or you can use `https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach`

Could also use single loop with `navSearch.value === '' ? 'block' : 'none'` here.